DeviceTests/DeviceTests.iOS/Info.plist

```diff
@@ -10,7 +10,6 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
-		<string>UIInterfaceOrientationPortraitUpsideDown</string>
```

Comment:
Can we add this back in, just to remove it from the diff with what's in the main repo

Reply:
I am unsure how i even removed that, but added back